# Joint Efforts to be set
string[] joint_names           # joint to apply wrench (linear force and torque)
float64[] effort              # effort to apply (Gets applied till next message changes it)
